recipes/des-desmeds/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "des-desmeds" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/esheldon/desmeds/archive/v{{ version }}.tar.gz
+  sha256: 7665af58beae73469c4b8e07b60a9cefb81f1b956ac03a810ddcc2d0bb5d42ad
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - fitsio
+    - des-easyaccess
+    - meds
+    - esutil
+    - pyyaml
+    - numpy
+
+test:
+  imports:
+    - desmeds
+
+about:
+  home: https://github.com/esheldon/desmeds
+  license: GPL-3
+  license_family: GPL
+  license_file: LICENSE
```

Comment:
We need to add the license to the repo and cut a new release for desmeds.

Reply:
done